gulpfile.js/util/pattern-library/macros/markup.html

```diff
@@ -0,0 +1,5 @@
+{% macro markup(file) %}
+```
```

Comment:
I was thinking if we want to wrap this in a `div`, depends if we plan to add a label in the top-left corner like for `example` blocks

Reply:
If we do we can do it later. Right now we don't. That shouldn't hold up this PR.